### ISSUE TITLE
DLLs not working with current Notepad++ versions

### ISSUE THREAD

**Issue by Progman2002** [jrebacz/NppGraphViz#7]
It looks like there was an update in the plugin system of Notepad++. The provided DLLs doesn't work anymore. Is it possible to update the DLLs to the current plugin system of Notepad++?

**Comment by jrebacz** [comment]
Thanks a lot for reporting this.  When I investigated on my 32-bit installation, I confirmed that the plugin was not working.  Specifically, it was not present in the "Plugins" drop-down from the file menu.  From what I see on the Notepad++ website, it looks like plugins are supposed to go in folders now.  I was able to get the plugin to work simply by making a folder in the plugin directory and moving the dll into it.  So, now the full path of the NppGraphViz plugin on my system is: "C:\Program Files (x86)\Notepad++\plugins\NppGraphViz\NppGraphViz.dll".  If you have 64-bit Notepad++ installed then the path will be different.  I'm going to go ahead and update the installation instructions.
